dockerfiles/che/rhel.Dockerfile

```diff
@@ -31,7 +31,7 @@ RUN mkdir /logs /data && \
     chmod -R g+rwX /home/user && \
     find /home/user -type d -exec chmod 777 {} \; && \
     java -version && echo -n "Server startup script in: " && \
-    find /home/user/codeready -name catalina.sh | grep -z /home/user/codeready/tomcat/bin/catalina.sh
+    find /home/user/eclipse-che -name catalina.sh | grep -z /home/user/eclipse-che/tomcat/bin/catalina.sh
```

Comment:
downstream we use codeready, not eclipse-che. If we make this change in upstream version we'll have to transform it in downstream jenkinsfile/sync job.

Reply:
anyway, codeready name shouldn't be in upstream

Reply:
true - see merged https://github.com/eclipse/che/pull/18679 which moves to `eclipse-che` in this file, and then we re-transform it back to `codeready` in the midstream and downstream versions.